app/controllers/npq_separation/admin/applications/reviews_controller.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module NpqSeparation
+  module Admin
+    module Applications
+      class ReviewsController < NpqSeparation::AdminController
+        helper_method :employment_types
+
+        def index
+          applications = Application.includes(:private_childcare_provider, :school, :user)
+                                    .merge(review_scope)
+                                    .merge(filter_scope)
+                                    .merge(search_scope)
+
+          @pagy, @applications = pagy(applications, limit: 9)
+        end
+
+      private
+
+        def employment_types
```

Comment:
should this array be on the `Application` model?

Reply:
@rwrrll Feel free to write a ticket - maybe schedule it for a couple of sprints time and we can evaluate in sprint review whether we have adequate capacity to do with it

Put it under uptime and maintenance epic, thanks